juju/juju
# Add support for juju list-operations

### Checklist


New API but list-operation sis behind a jujuv3 feature flag.

----

## Description of change

Add support for a revamped list-operations CLI.
We now return operations results, which may each have matching tasks (actions). Previously we were returning a list of tasks (actions) directly. A bunch of old action specific query code in the facade is removed and replaced by a direct state call to fetch the operations.
For future use, add offset/limit params so the CLI can batch possibly large results.

## QA steps

Run some actions.
Run list-operations (with/without --format yaml) to see that the selected operations and any child tasks are returned.
